tooling/nargo_cli/src/cli/prove_cmd.rs

```diff
@@ -125,11 +125,12 @@ pub(crate) fn prove_package(
     // Get the entry point witness for the ABI
     let main_witness =
         &witness_stack.peek().expect("Should have at least one witness on the stack").witness;
-    let public_abi = compiled_program.abi.public_abi();
-    let (public_inputs, return_value) = public_abi.decode(main_witness)?;
+    let abi = compiled_program.abi;
+    let (inputs, return_value) = abi.decode(main_witness)?;
```

Comment:
Naming: `all_inputs` might have been even better, but not a blocker.

Reply:
Done.